src/AggregateFunctions/AggregateFunctionSparkbar.cpp

```diff
@@ -378,6 +378,7 @@ AggregateFunctionPtr createAggregateFunctionSparkbar(const std::string & name, c
 void registerAggregateFunctionSparkbar(AggregateFunctionFactory & factory)
 {
     factory.registerFunction("sparkbar", createAggregateFunctionSparkbar);
```

Comment:
I think it's better to simply just do:
```suggestion
    factory.registerFunction("sparkbar", createAggregateFunctionSparkbar, AggregateFunctionFactory::CaseInsensitive);
```
In this way, it will cover additional cases too like `sparkBar`, `SparkBar`, 'Sparkbar' etc.

Reply:
@bharatnc I'm not sure covering all situations is what they need.